src/pages/JobIndex/JobIndex.tsx

```diff
@@ -14,16 +15,23 @@ const JobIndex = () => {
     method: 'get',
     withCredentials: authEnabled,
   })
+  const navigate = useNavigate()
 
   useEffect(() => {
     if (data && !error) {
       setJobs(data)
     }
   }, [data, error])
 
+  // temporary for demo
```

Comment:
This whole page is being refactored under a different ticket. This is only here to provide a way to get to a page that falls under this ticket.